src/components/buttons/CancelButton/CancelButton.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { Button } from 'semantic-ui-react';
 import type { ButtonProps } from 'semantic-ui-react';
+import { Button } from 'semantic-ui-react';
+
+export type CancleButtonProps = ButtonProps;
 
 /**
  * CancelButton is a component to present cancellation button in Modal component.
  *
  * CancelButton is customized version of [Semantic UI-React's Button component](https://react.semantic-ui.com/elements/button),
  * so all properties of that component (eg. onClick, disabled, ...) can be used here.
  */
-export default function CancelButton({
+export function CancelButton({
     content = 'Cancel',
     icon = 'remove',
     className = 'cancel',
     ...buttonProps
-}: ButtonProps) {
+}: CancleButtonProps) {
```

Comment:
typo CancleButtonProps
```suggestion
}: CancelButtonProps) {
```